src/mongocxx/test/CMakeLists.txt

```diff
@@ -262,6 +262,8 @@ set_tests_properties(unified_format_spec PROPERTIES
   ENVIRONMENT "UNIFIED_FORMAT_TESTS_PATH=${PROJECT_SOURCE_DIR}/../../data/unified-format")
```

Comment:
Stopping here for now, I'll continue into the tests once this first batch of comments is addressed.